templates/calculatedsummary.html

```diff
@@ -1,34 +1,5 @@
-{% extends 'layouts/_questionnaire.html' %}
-{% from "components/button/_macro.njk" import onsButton %}
-{% from "components/panel/_macro.njk" import onsPanel %}
+{% extends 'layouts/_calculatedsummary.html' %}
 
-{% set save_on_signout = true %}
-
-{% block form_content %}
-  <h1 data-qa="calculated-summary-title">{{content.summary.title}}</h1>
-
-  {% call onsPanel({
-    "classes": "ons-u-mb-l"
-  }) %}
-    <p>{{ _("Please review your answers and confirm these are correct") }}</p>
-  {% endcall %}
-
-  <div class="ons-u-mb-m">
-    {% include 'partials/summary/summary.html' %}
-  </div>
-{% endblock -%}
-
-{% block submit_button %}
-  {{
-    onsButton({
-      "text": _("Yes, I confirm these are correct"),
-      "variants": 'timer',
-      "attributes": {
-        "data-qa": "btn-submit",
-        "data-ga-category": "Submit button",
-        "data-ga-action": "Confirm",
-        "data-ga-label": "Confirm button click"
-      }
-    })
-  }}
+{% block form_title %}
+  <h1 data-qa="calculated-summary-title">{{ content.summary.title }}</h1>
```

Comment:
We could possibly use one template and use existing `content.summary.summary_type` for the logic but then, is using one template better?

Reply:
the summary type would be CalculatedSummary or GrandCalculatedSummary so it might be a bit ugly to get the data-qa that way, unless theres another one?

Reply:
@petechd  happy for this to stay as is?